transcription/doc_intelligence.py

```
@@ -0,0 +1,113 @@
+def run_doc_intell_pipeline(image_path: str):
+    # Check if the file is an image 
+    if image_path.lower().endswith((".png", ".jpg", ".jpeg")):
+        # Processing an image
+        image_result = process_image(image_path)
+        _, confidence_text = get_image_words(image_result)
+        document_content = image_result.content + "\n\nConfidence Metrics:\n" + confidence_text 
+        extracted_info = extract_info(document_content, example_result, example_output)
+
+        try:
+            # Formatting results
+            data = eval(extracted_info) 
```

Ruff (0.14.8)

Use of possibly insecure function; consider using `ast.literal_eval`

(S307)